React/wbooks/src/Components/BookCover.js

```diff
@@ -3,19 +3,20 @@ import PropTypes from 'prop-types';
 
 import defaultImg from '../assets/default_book.svg';
 
-function BookCover({ imageUrl, imgStyle }) {
+function BookCover({ imageUrl, imgStyle, defaultStyle }) {
   return imageUrl ? (
     <img className={imgStyle} alt="Book cover" src={imageUrl} />
   ) : (
     <div className={`${imgStyle} book-default-background`}>
-      <img alt="Default book cover" src={defaultImg} />
+      <img className={defaultStyle} alt="Default book cover" src={defaultImg} />
```

Comment:
Alts are better if they explain what you should be looking at. If you can, pass the name of the book as alt. Also, you shouldn't clarify if it's the default cover or not.